### PR TITLE
Migrate ErrorFormTooltip to Tailwind

### PR DESCRIPTION
**Pull request by DSil**
[FEPLT-1722](https://kiwicom.atlassian.net/browse/FEPLT-1722)
 Storybook: https://orbit-mainframev-migrate-error-form-tooltip-tailwind.surge.sh

### CODE REVIEW

**Comment by mainframev** [comment]
The position of tooltip is slightly off on RTL (in master in one side, in this branch in another) but still it's a 🐛

File: packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx
Comment (mainframev) [inline packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx:142]: We have to be mindful about those [&>.class] selectors. For instance, here it won't work for all text-links, if the user decides to pass more than one text-link, it will apply only to the first one

File: packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx
Comment (mainframev) [inline packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx:131]:
```suggestion
          "absolute start-xs",
```

File: packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx
Comment (mainframev) [inline packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx:132]:
```suggestion
          isVertical ? "bottom-xxxs" : "top-xxxs",
```

File: packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx
Comment (mainframev) [inline packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx:135]:
```suggestion
          "before:absolute before:h-xs before:w-xs before:-translate-x-1/2 before:-translate-y-1/2 before:rotate-45",
```

**Review by mainframev — CHANGES_REQUESTED** [review]

File: packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx
Comment (mainframev) [reply packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx:142]: Should be "[&_a]:text-white-normal"

File: packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx
Comment (oreqizer) [reply packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx:96]: `attrs.popper?["data-popper-placement"] === "top-end"`

Comment (DSil) [reply packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx:96]: @oreqizer suggestion did not work. Was throwing error
```
This comparison appears to be unintentional because the types 'string[]' and 'string' have no overlap.ts(2367)
This condition will always return 'false' since JavaScript compares objects by reference, not value.ts(2839)
```

Ended up doing
```js
const isVertical =
  attrs.popper &&
  (attrs.popper["data-popper-placement"] === "top-start" ||
    attrs.popper["data-popper-placement"] === "top-end")
    ? "bottom"
    : "top";
```

File: packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx
Comment (DSil) [reply packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx:142]: Right, my bad, thanks

**Comment by DSil** [comment]
@mainframev regarding the position, it's funny because I was checking with dev tools open and it was working fine. Turns out you checked it without dev tools and saw it buggy. Why? Because the FormLabel element has a different padding on desktop and above. With dev tools open, the viewport was of size tablet, and so I was unable to spot the bug, and so I only fixed it for the other one that we had in Master… But anyway, it's fixed now, thanks 👍

**Review by mainframev — APPROVED** [review]

File: packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx
Comment (oreqizer) [reply packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx]: 🤔 really? that's weird, this didn't work?

```ts
const isVertical =
    attrs.popper?.["data-popper-placement"] === "top-start" ||
    attrs.popper?.["data-popper-placement"] === "top-end"
      ? "bottom"
      : "top";
```

I mean it is just syntax here since we're not enforcing [prefer nullish coalescing](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/prefer-nullish-coalescing.md) in ESlint, so no problem, I just find it interesting

File: packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx
Comment (mainframev) [inline packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx:141]: do we need these focus and hover classes if the color is not changing?

Comment (mainframev) [reply packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx:141]: besides that nothing. LGTM and can be merged :) @oreqizer I though we were waiting for you approval on that also :)

Comment (DSil) [reply packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx:141]: I believe it is here to enforce the color. If we use the Text inside, the default colors of the text are given by classes that will always be overwritten by this. It's true, we don't seem to have styles on hover which means we could probably remove it. I would personally not risk it at this moment tbh. But fair question

**Review by oreqizer — APPROVED** [review]
lgtm 👍

**Pull request merged**
